### ISSUE TITLE
Support for justify-content-center?

### ISSUE THREAD

**Issue by actongorton** [dmhendricks/bootstrap-grid-css#1]
Hi, was just reading through the code and didn't see support for `justify-content-center` -- am I missing it?

**Comment by giovannipds** [comment]
Agree. There's no `align-items-center` as well... it **must have** these features for yesterday (note: anyone can do it).
